The Learning Servlet

omniprof opened the pull request:
I must admit that this is my first open source pull request. It adds the jakartaee-examples my LearningServlet. It uses package names consistent with all other projects, only java.util.logging used for logging and a basic test that verifies that the project will run on a server that is modeled after tests found in other modules. The project is now part of the WebServlets modules. It is extensively commented so that a developer at any level can understand the basic components of a JakartaEE Servlet. The Learning Servlet will work with Java 1.8 or Java 11. 

In addition I corrected minor problem with top level pom where a tag ended with >> rather than >

arjantijms commented on servlet/theLearningServlet/src/test/java/jakartaee/examples/servlet/thelearningservlet/TheLearningServletTest.java:
```diff
@@ -0,0 +1,96 @@
+/**
+ * The JUnit tests for the Servlet API @WebServlet example.
+ *
+ * @author Manfred Riem (mriem@manorrock.com)
```
Is this correct? :P

omniprof replied:
There are two issues I will resolve. The easier one is to expand the test to return to the index.html page and press the second button. The second one has to do with the email address that I inadvertently used to sign the submission. As I can edit the config locally I'm hoping that when I push it to my repo it will now have the correct user name and email address.

arjantijms commented on servlet/theLearningServlet/src/test/java/jakartaee/examples/servlet/thelearningservlet/TheLearningServletTest.java:
```diff
@@ -0,0 +1,96 @@
+     */
+    @RunAsClient
+    @Test
+    public void testWebServlet() throws Exception {
```
You'd probably want to be more specific here with the method name. What is it exactly testing? The Servlet exposes many operations, which one does this method test?

omniprof commented:
OK, I think I have it. Otherwise I should cancel the PR and resubmit fresh. I have fixed the owner and added the signed-off-by but it took three commits and 3 pull requests. Advise and I will take care of it. I apologize for errors but once i get this right the only issues should be with my code and not my process.

omniprof commented:
Going to try again.
